### PR TITLE
Const modifier

### PR DESCRIPTION
**Pull request by ramico**
this shall pop-up code

### CODE REVIEW

File: includes/class-wc-siftscience-options.php [hunk 1]
```diff
@@ -13,21 +13,21 @@
...
-		public static $min_order_value = 'siftsci_min_order_value';
-		public static $stats_api = 'https://sift.fermiac.staat.us';
+		const API_KEY 					= 'siftsci_api_key';
+		const JS_KEY 					= 'siftsci_js_key';
+		const NAME_PREFIX 				= 'siftsci_name_prefix';
+		const THRESHOLD_GOOD 			= 'siftsci_threshold_good';
+		const THRESHOLD_BAD 			= 'siftsci_threshold_bad';
+		const SEND_ON_CREATE_ENABLED 	= 'siftsci_send_on_create_enabled';
+		const MIN_ORDER_VALUE 			= 'siftsci_min_order_value';
+		public static $mode 			= 'siftsci_reporting_mode';
```
Comment (nabsul) [inline includes/class-wc-siftscience-options.php:23]: Why did you only change half of them?

Comment (ramico) [reply includes/class-wc-siftscience-options.php:23]: want your approval that you like that write do it for me please

Comment (nabsul) [reply includes/class-wc-siftscience-options.php:23]: Sure, it's a good change. Just two comments:

- The area between the const name and the `=` sign should only have spaces. No tabs.
- Please define all the constants to be public explicitly.

Comment (ramico) [reply includes/class-wc-siftscience-options.php:23]: constants are public 
I'll write it ~OK

**Comment by ramico** [comment]
All found renamed constant values are used with either 

- get_option

- update option 

which in turns doesn't change the constant value rather than the option 
ps: separation was made to get an easier hang of things

File: includes/class-wc-siftscience-options.php [hunk 2]
```diff
@@ -13,21 +13,21 @@
...
-		public static $stats = 'siftsci_stats';
-		public static $stats_last_sent = 'siftsci_stats_last_sent';
-		public static $min_order_value = 'siftsci_min_order_value';
-		public static $stats_api = 'https://sift.fermiac.staat.us';
+		const API_KEY 					= 'siftsci_api_key';
+		const JS_KEY 					= 'siftsci_js_key';
+		const NAME_PREFIX 				= 'siftsci_name_prefix';
+		const THRESHOLD_GOOD 			= 'siftsci_threshold_good';
+		const THRESHOLD_BAD 			= 'siftsci_threshold_bad';
+		const SEND_ON_CREATE_ENABLED 	= 'siftsci_send_on_create_enabled';
```
Comment (ramico) [inline includes/class-wc-siftscience-options.php]: maybe it's is better to just call it AUTO_SEND_CHECKED

File: includes/class-wc-siftscience-options.php [hunk 3]
```diff
@@ -13,28 +13,27 @@
...
-		public static $send_stats = 'siftsci_send_stats';
-		public static $threshold_good = 'siftsci_threshold_good';
-		public static $threshold_bad = 'siftsci_threshold_bad';
-		public static $log_level_key = 'siftsci_log_level';
-		public static $guid = 'siftsci_guid';
-		public static $stats = 'siftsci_stats';
-		public static $stats_last_sent = 'siftsci_stats_last_sent';
-		public static $min_order_value = 'siftsci_min_order_value';
-		public static $stats_api = 'https://sift.fermiac.staat.us';
+		const API_KEY = 'siftsci_api_key';
```
Comment (nabsul) [inline includes/class-wc-siftscience-options.php:16]: please make all public

**Pull request merged**